examples/hosts/app-integration/external-data/src/view/debugView.tsx

```diff
@@ -5,17 +5,14 @@
 
 import React, { useEffect, useState } from "react";
 import isEqual from 'lodash.isequal'
-import type { IAppModel, TaskData } from "../model-interface";
+import type { TaskData } from "../model-interface";
 import { customerServicePort } from "../mock-service-interface";
 
 /**
  * {@link DebugView} input props.
  */
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
What's the need for the empty interfaces?

Reply:
I suppose we could remove them!